ci/deps/actions-311.yaml

```
@@ -18,7 +18,7 @@ dependencies:
 
   # required dependencies
   - python-dateutil
-  - numpy<1.24.0
+  - numpy>=1.24.0
```

```suggestion
  - numpy
```

(Until we enforce the same min version on these files)